Umbraco-Cloud/index.md

```diff
@@ -7,37 +7,53 @@ versionFrom: 7.0.0
 
 Welcome to the Umbraco Cloud documentation. Here you can read all about how to set up and work with Umbraco Cloud. You can find articles about the deployment workflow, how to work locally and all the features that comes with your Umbraco Cloud project. You can also find articles about troubleshooting various issues.
 
+This is some tet, because we're testing how to create Pull Requests.
+
+Lala.
+
+These are some more lines.
```

Comment:
This is really cool!
But, how about we do this instead:

```suggestion
```

What do you think?